SHARE-159 Show number of rejected programs/comments per user

wolfianer opened the pull request:
Implemented "Reported Users" in the admin panel.

### Short Description
Implemented "Reported Users" in the admin panel.

### Tests - additional information
As written in discord some behat tests fail for the admin panel, because I added the working Assert::assert() function instead of the not working assert() function. 

[FailedTests.txt](https://github.com/Catrobat/Catroweb-Symfony/files/4191967/FailedTests.txt)

dmetzner commented on src/Entity/User.php:
```diff
@@ -452,4 +464,46 @@ public function isFollowing($user)
+  /**
+   * @return mixed
+   */
+  public function  getProgramInappropriateReports()
```
2 whitespaces

dmetzner commented on src/Entity/User.php:
```diff
@@ -452,4 +464,46 @@ public function isFollowing($user)
+  /**
+   * @return mixed
+   */
+  public function  getComments()
```
again

dmetzner reviewed (changes requested):
Nice PR, works as it should, only two minor things:

* fix the mapping errors (can be found in the symfony profiler) 

![image](https://user-images.githubusercontent.com/40868718/74339320-280ae780-4da4-11ea-9fde-825052f1ef8e.png)

* might re-format User.php. There are two necessary whitespaces.

Tests Are still running. I will update this comment when they are done.

dmetzner commented:
perfect, now just squash it, then we can merge it

dmetzner committed:
SHARE-159 Show number of rejected programs/comments per user

- Implementing "Reported Users" in the admin panel.
- fixing old assert calls that never worked -> now we have some admin files
- fixing usercomments/user mapping
- adding wait to be visible (to fix image upload tests)
